agent/config/config.go

```diff
@@ -101,6 +109,7 @@ type Config struct {
 	BindAddr                         *string                  `json:"bind_addr,omitempty" hcl:"bind_addr" mapstructure:"bind_addr"`
 	Bootstrap                        *bool                    `json:"bootstrap,omitempty" hcl:"bootstrap" mapstructure:"bootstrap"`
 	BootstrapExpect                  *int                     `json:"bootstrap_expect,omitempty" hcl:"bootstrap_expect" mapstructure:"bootstrap_expect"`
+	CacheConfig                      CacheConfig              `json:"cache,omitempty" hcl:"cache" mapstructure:"cache"`
```

Comment:
```suggestion
	Cache                      Cache              `json:"cache,omitempty" hcl:"cache" mapstructure:"cache"`
```

(although this change will need a `gofmt`)

Reply:
This is in conflict with Cache defined in configuration:
```
agent/config/runtime.go:26:6: Cache redeclared in this block
	previous declaration at agent/config/config.go:62:6
```
(the one defined in agent/config/config.go has optional fields EntryFetchMaxBurst/EntryFetchRateLimit while those are mandatory for CacheConfig).

I could re-use the same structure, but in that case EntryFetchMaxBurst and EntryFetchRateLimit would be pointers, not values... it is what you want?

Reply:
Ah, right! I forgot there are these 2 structs in `config`. I wonder if we can use `agent/cache.Options` in `RuntimeConfig` , and use `Cache` in `Config`. That way we don't need to translate it again. We can use the value directly from `RuntimeConfig` in  `cache.New()`

Reply:
Done in 48e50fe1a119b0b9480eb7c81154ff44a517d9f9